playbooks/hc-ansible-deployment/tasks/gluster_deployment.yml

```diff
@@ -45,6 +45,27 @@
           - "disk_size.stdout|int > gluster_features_min_disk|default(15000)"
         fail_msg: "The size of /var should be greater than or equal to 15G"
       when: gluster_features_force_varlogsizecheck | default(true)
+      
+      
+    - name: Check if block device is 512B
```

Comment:
this check is needed again because i wanted to get a new list of boolean , to proceed with further checks

Reply:
gluster_features_force_varlogsizecheck is default false in rhhi deployment, Is this check necessary to proceed with 512B check?

Reply:
no the intention of this PR is to not proceed with the further task if the user uses a mix of 4096 and 512 blocks for volumes,

i believe gluster_features_force_varlogsizecheck is to check the /var/log size, 
in that case, gluster_features_force_varlogsizecheck doesnt matter if its true or false.